packages/launcher/electron-builder.json

```diff
@@ -1,15 +1,40 @@
 {
   "appId": "com.mainframe.launcher",
-  "productName": "Mainframe",
+  "productName": "MainframeOS",
   "artifactName": "${productName}.${ext}",
   "appImage": {
     "systemIntegration": "doNotAsk"
   },
+  "linux": {
+    "category": "Network",
+    "target": ["AppImage", "dir"],
+    "icon": "./build/icon.png"
```

Comment:
Could be window manager dependant, but I didn't see an icon in the linux build

Reply:
this is interesting, I'll play with it